jquery.sumoselect.js

```diff
@@ -513,7 +518,7 @@
                     var O = this;
                     O.placeholder = "";
                     if (O.is_multi) {
-                        var sels = O.E.find(':selected').not(':disabled'); //selected options.
+                        var sels = O.E.find(':selected').not(':disabled').not(':hidden'); //selected options.

```

Comment:
There are a lot of other places that list selected options by checking if they are not `:disabled`, and would need to check for hidden too.